librz/core/rop.c

```diff
@@ -214,12 +260,12 @@ static bool rz_rop_print_json_mode(const RzCore *core, const RzCoreAsmHit *hit,
 		return false;
 	}
 
-	pj_o(pj);
-	pj_kn(pj, "offset", hit->addr);
-	pj_ki(pj, "size", hit->len);
-	pj_ks(pj, "opcode", rz_asm_op_get_asm(asmop));
-	pj_ks(pj, "type", rz_analysis_optype_to_string(aop.type));
-	pj_end(pj);
+	pj_o(context->state->d.pj);
```

Comment:
You could just get `pj` pointer once and keep the rest of the old code unchanged

Reply:
And this one

Reply:
Done :+1: